pkg/controllers/scheduling/clusterset_event_handler.go

```diff
@@ -0,0 +1,64 @@
+package scheduling
+
+import (
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/klog/v2"
+
+	clusterlisterv1alpha1 "github.com/open-cluster-management/api/client/cluster/listers/cluster/v1alpha1"
+	clusterapiv1alpha1 "github.com/open-cluster-management/api/cluster/v1alpha1"
+	errorhelpers "github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+type clusterSetEventHandler struct {
+	clusterSetBindingLister clusterlisterv1alpha1.ManagedClusterSetBindingLister
+	placementLister         clusterlisterv1alpha1.PlacementLister
+	enqueuePlacementFunc    enqueuePlacementFunc
+}
+
+func (h *clusterSetEventHandler) OnAdd(obj interface{}) {
+	// ignore Add event
+}
+
+func (h *clusterSetEventHandler) OnUpdate(oldObj, newObj interface{}) {
+	// ignore Update event
+}
+
+func (h *clusterSetEventHandler) OnDelete(obj interface{}) {
+	// skip resource other than ManagedClusterSet
+	clusterSet, ok := obj.(*clusterapiv1alpha1.ManagedClusterSet)
```

Comment:
i think you would also need to handle the resource with partial objectmeta as here https://github.com/openshift/library-go/blob/7392ea9b02ca2e4a13cf002cdd18485a99857640/pkg/controller/factory/controller_context.go#L69-L71

Reply:
Done.